packages/vite/src/node/server/http.ts

```diff
@@ -133,3 +133,29 @@ async function createCertificate() {
   })
   return pems.private + pems.cert
 }
+
+async function getCertificate(config: ResolvedConfig) {
+  if (!config.cacheDir) return await createCertificate()
+
+  const cachePath = path.join(config.cacheDir, '_cert.pem')
+
+  try {
+    const [stat, content] = await Promise.all([
+      fsp.stat(cachePath),
+      fsp.readFile(cachePath, 'utf8')
+    ])
+
+    if (Date.now() - stat.ctime.valueOf() > 30 * 24 * 60 * 60 * 1000) {
+      throw new Error('cache is outdated.')
+    }
+
+    return content
+  } catch (e) {
```

Comment:
The error is never accessed :thinking: I think we can omit it

```suggestion
  } catch {
```

Reply:
Yes, you are right. But I just fllow you:
https://github.com/vitejs/vite/blob/5c177db9597344b8151c8e421ddbced36c4151c7/packages/vite/src/node/build.ts#L374